specification/communication/Communication.Messages/models.tsp

```diff
@@ -205,6 +213,28 @@ model AudioNotificationContent extends NotificationContent {
   ...MediaNotificationContentBase;
 }
 
+@added(Versions.c2024_11_15)
+@doc("A request to send a Reaction notification.")
+model ReactionNotificationContent extends NotificationContent {
+  @doc("Message notification type is reaction.")
+  kind: CommunicationMessageKind.reaction;
+
+  @doc("emoji content like \\uD83D\\uDE00.")
+  emoji?: string;
+
+  @doc("ID of the previous message you want to reply to.")
+  messageId?: string;
```

Comment:
nit, why is `emoji` optional?

What happens when user send this without `emoji` nor `messageId`?

Reply:
Updating this.